test/src/400-ducc-ingest_images/main

```diff
@@ -82,16 +90,16 @@ input:
     - 'https://registry.hub.docker.com/library/centos:centos6'
 EOL
     printf "done\n"
-    
+
     # set the password to access the docker hub
-    export DOCKER2CVMFS_DOCKER_REGISTRY_PASS=mock_pass
-    
+    export DUCC_DOCKER_REGISTRY_PASS=mock_pass
+
     # crete the repository where to store the content
-    printf "Creating CVMFS repo..."
-    cvmfs_server mkfs -o $USER $repository >> /dev/null
+    printf "*** Creating CVMFS repo..."
+    create_empty_repo $repository $USER || return $?
     printf "done\n"
-    
-    printf "Changing configuration to read the new repo..."
+
+    printf "*** Changing configuration to read the new repo..."
```

Comment:
The SERVER_URL on localhost won't work for the S3 tests.  Have a look into the helper functions, there should be one to get the stratum 0 URL